react-basic-practice/src/App.tsx

```diff
@@ -2,14 +2,19 @@ import React from "react";
 import { ChakraProvider } from "@chakra-ui/react";
 import { theme } from "themes";
 import Button from "@components/Button";
+import Search from "@components/Search";
+import Input from "@components/Input";
```

Comment:
create index.ts file contains all components. After that, we just need import from `@component` and we can get all component with only one import

Reply:
I only imported to see the UI, but I understand and will update that later, but currently it is not certain that these components are imported here.

Reply:
I don't think so, everything should be clear now. Pls update